src/apache/conf/ssl.conf

```diff
@@ -40,8 +40,7 @@ LoadModule ssl_module modules/mod_ssl.so
 #   Disable SSLv3 by default (cf. RFC 7525 3.1.1).  TLSv1 (1.0) should be
 #   disabled as quickly as practical.  By the end of 2016, only the TLSv1.2
 #   protocol or later should remain in use.
-SSLProtocol all -SSLv3
-SSLProxyProtocol all -SSLv3
+SSLProtocol All -SSLv3 -TLSv1 -TLSv1.1
```

Comment:
The only difference in the ssllabs score between the current config and this PR is the protocol support, which I assume it due to this line. I bet if this is the only change you make you'd get the same score you just got. Unless there's a reason to change the suite, I say leave the rest of the config alone and only make this change.